src/deterministic_vi/elbo.jl

```diff
@@ -235,11 +235,12 @@ function accum_star_pos!{NumType <: Number}(
                     x::Vector{Float64},
                     wcs_jacobian::Array{Float64, 2},
                     calculate_derivs::Bool)
+
     eval_bvn_pdf!(elbo_vars.bvn_derivs, bmc, x)
 
-    # TODO: Also make a version that doesn't calculate any derivatives
-    # if the object isn't in active_sources.
-    get_bvn_derivs!(elbo_vars.bvn_derivs, bmc, true, false)
+    if elbo_vars.calculate_derivs && calculate_derivs
```

Comment:
"calculate_derivs" means what it says.  As used now, we determine whether to calculate_derivs based on whether it is an active source.  That's not necessarily the case.

This is the way it's always been.  Why do you want to change it now?